Bot/Logger.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import os
+
+class LoggingFormatter(logging.Formatter):
+    # Colors
+    black = "\x1b[30m"
+    red = "\x1b[31m"
+    green = "\x1b[32m"
+    yellow = "\x1b[33m"
+    blue = "\x1b[34m"
+    gray = "\x1b[38m"
+    # Styles
+    reset = "\x1b[0m"
+    bold = "\x1b[1m"
+
+    COLORS = {
+        logging.DEBUG: gray + bold,
+        logging.INFO: blue + bold,
+        logging.WARNING: yellow + bold,
+        logging.ERROR: red,
+        logging.CRITICAL: red + bold,
+    }
+
+    def format(self, record):
+        log_color = self.COLORS[record.levelno]
+        format = "(black){asctime}(reset) (levelcolor){levelname}(reset) (green){name}(reset) {message}"
+        format = format.replace("(black)", self.black + self.bold)
+        format = format.replace("(reset)", self.reset)
+        format = format.replace("(levelcolor)", log_color)
+        format = format.replace("(green)", self.green + self.bold)
```

Comment:
```suggestion
        format = "{black}{{asctime}}{reset} {levelcolor}{{levelname}}{reset} {green}{{name}}{reset} {{message}}".format(
            black= self.black + self.bold,
            reset = self.reset,
            levelcolor=log_color,
            green=self.green+self.bold,
        )
```

We can make use of `{{` and `}}` to escape the `{` and `}` so that we can format them later.